### PR TITLE
fix default value when creating new date or time company custom field

### LINKED ISSUE #9512: Attempted to call an undefined method named "validateValue" of class "Symfony\Component\Validator\Validator\RecursiveValidator"
Mautic version - v3.2.1
PHP version - 7.3.24 

## Description:
Error when trying to create a new custom field of type Time (and I believe Date).

### Steps to reproduce:
1. Create a new custom field for company that is a time field (I believe date will do this as well)
2. Populate the values and try to save the new custom field, below error will be given
 
### Log errors: 

mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "validateValue" of class "Symfony\Component\Validator\Validator\RecursiveValidator"." at mautic\app\bundles\LeadBundle\Form\Type\FieldType.php line 354 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"validateValue\" of class \"Symfony\\Component\\Validator\\Validator\\RecursiveValidator\". at mautic\\app\\bundles\\LeadBundle\\Form\\Type\\FieldType.php:354)"} []

### Fix:
I am unable to fix this right now, but it seems to be an easy fix locally.  I believe validateValue has been changed to validate.  This need to be updated in two places in LeadBundle\Form\Type\FieldType.php  Lines 337 and 354.

Old code:
$violations = $validator->validateValue(...

New code:
$violations = $validator->validate(...

### PR DESCRIPTION
**Pull request by johbuch**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #9512  

#### Description:
Fix the bug that displays when you add a default value while creating a new company custom (date or time) field
#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new company time field
3. add a default value
4. Save and see that it works

### CODE REVIEW

**Comment by npracht** [comment]
Hi @johbuch can you redo the PR against 3.2 branch ?
When you create your own branch to create your fix, use `git branch fix-dateandtime-defaultvalue 3.2` then do your PR

File: app/bundles/LeadBundle/Form/Type/FieldType.php [hunk 1]
```diff
@@ -37,14 +37,8 @@
 
 class FieldType extends AbstractType
 {
-    /**
```
Comment (kuzmany) [inline app/bundles/LeadBundle/Form/Type/FieldType.php]: Both annotations shouldn't be removed. Probably was removed on windows by cs-fixer (it's bug). But need get back

**Review by kuzmany — CHANGES_REQUESTED** [review]

File: app/bundles/LeadBundle/Form/Type/FieldType.php (hunk 1 above)
Comment (johbuch) [reply app/bundles/LeadBundle/Form/Type/FieldType.php]: Hi @kuzmany indeed its weird i saw that but I didn't remove it on purpose. I was planning to adding it again then.
anyway, i will fix it

**Review by kuzmany — DISMISSED** [review]
Works for me. Thanks 👍

**Comment by npracht** [comment]
closed in favour of https://github.com/mautic/mautic/pull/9534

**Pull request closed without merging**